sdk/nodejs/README.md

```diff
@@ -1,17 +1,4 @@
-The Google Cloud Platform (GCP) provider package offers support for all GCP services and their properties.
-Services are exposed as types from modules such as `compute`, `container`, `sql`, and `storage`,
-among many others. Using this package allows you to programmatically declare instances of any GCP
-services using infrastructure as code, which Pulumi then uses to drive the GCP API.
-
-If this is your first time using this package, these two resources may be helpful:
-
-* [GCP Getting Started Guide](https://www.pulumi.com/docs/get-started/gcp/): Get up and running quickly.
-* [GCP Pulumi Setup Documentation](https://www.pulumi.com/docs/get-started/gcp/configure/): How to configure Pulumi
-    for use with your GCP account.
-
-Use the navigation below to see detailed documentation, including sample code, for each of the supported GCP services.
-
-> This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-google)
```

Comment:
We shouldn't be losing this text--see https://github.com/pulumi/pulumi-azuread/pull/78#discussion_r435499634